requests_gssapi/gssapi_.py

```diff
@@ -83,13 +83,13 @@ class HTTPSPNEGOAuth(AuthBase):
     def __init__(self, mutual_authentication=REQUIRED, service="HTTP",
                  delegate=False, force_preemptive=False, principal=None,
                  hostname_override=None, sanitize_mutual_error_response=True,
-                 creds=None):
+                 creds=None, opportunistic_auth=False):
```

Comment:
I was thinking we shouldn't retain force_preemptive, but provide a translation from that flag to opportunistic_auth in the wrapper that respondes to the old class name (HTTPKerberosAuth)